tutorials/inputs/handling_quit_requests.rst

```diff
@@ -60,11 +60,12 @@ at any time by either the user or the OS. A way to plan ahead for this
 possibility is to utilize ``NOTIFICATION_APPLICATION_PAUSED`` in order to 
 perform any needed actions as the app is being suspended.
 
+\ **Note:** On iOS, you only have approximately 5 seconds to finish a task started by this signal. If you go over this allotment, iOS will kill the app instead of pausing it.
```

Comment:
```suggestion
.. note:: On iOS, you only have approximately 5 seconds to finish a task started by this signal. If you go over this allotment, iOS will kill the app instead of pausing it.
```
Correct syntax